registry/client/repository.go

```diff
@@ -377,11 +377,7 @@ func (bs *blobs) Stat(ctx context.Context, dgst digest.Digest) (distribution.Des
 }
 
 func (bs *blobs) Get(ctx context.Context, dgst digest.Digest) ([]byte, error) {
-	desc, err := bs.Stat(ctx, dgst)
```

Comment:
@aaronlehmann This was fairly necessary when we had support for tarsum but is much less important now. I don't see any major issues with this change in the current environment.